sql/06_qna.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS qna (
+    id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    post_id INT NOT NULL,
+    comment_id INT NOT NULL,
+    FOREIGN KEY (post_id) REFERENCES posts(id) ON DELETE CASCADE,
+    FOREIGN KEY (comment_id) REFERENCES comments(id) ON DELETE CASCADE,
+    UNIQUE KEY (post_id, comment_id)
+);
```

Comment:
`post_id` 컬럼에만 `UNIQUE KEY` 제약조건이 걸려야 할 것 같습니다.
이미 채택을 완료한 게시글에서 다른 댓글도 채택 요청 가능하네요.

```suggestion
CREATE TABLE IF NOT EXISTS qna (
    id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
    post_id INT NOT NULL UNIQUE KEY,
    comment_id INT NOT NULL,
    FOREIGN KEY (post_id) REFERENCES posts(id) ON DELETE CASCADE,
    FOREIGN KEY (comment_id) REFERENCES comments(id) ON DELETE CASCADE
);
```

Reply:
오우 감사합니다!

일단 post_id, comment_id 개별로유니크 달아놨습니다!